lms/djangoapps/instructor/tests/test_proctoring.py

```diff
@@ -156,5 +156,5 @@ def _assert_proctoring_tab_available(self, available):
         """
         func = self.assertIn if available else self.assertNotIn
         response = self.client.get(self.url)
-        func(self.proctoring_link, response.content)
-        func('proctoring-wrapper', response.content)
+        func(self.proctoring_link, response.content.decode('utf-8'))
+        func('proctoring-wrapper', response.content.decode('utf-8'))
```

Comment:
Here again, I think we should switch `func` to be a reference to either `self.assertContains` or `self.assertNotContains` and just pass in the response object directly.

Reply:
@jmbowman doing that somehow fails in underlying `assertContains` method, it seems like it tries to assert the `http` status codes for the given `response`
```/root/edxapp_toxenv/py35-django111/lib/python3.5/site-packages/django/test/testcases.py:357: in _assert_contains
    " (expected %d)" % (response.status_code, status_code)
E   AssertionError: 404 != 200 : Couldn't retrieve content: Response code was 404 (expected 200)```

Reply:
Ok, yeah that's a problem for the "not in" case.

Reply:
we can pass the status code for comparison.

Reply:
to `assertNotContains`?